python/perf-kernels/flash-attention.py

```diff
@@ -17,6 +17,7 @@
 4) Multi and grouped query attention
 5) Variable sequence lengths
 6) ALiBi and matrix bias
+7) Persistent kernels (separated to attn_fwd_persistent)
```

Comment:
I think since this is a list of attention features, you can remove this from here. But we should keep it in the README as that is a general list of all features, not attention specific.